#velocity and angular velocity for motion center

uint8 Mode
uint8 Enable
float64 Vx
float64 Vy
float64 W
uint8 VW_Type
uint8 Cir_Flag
